src/SFA.DAS.Assessor.Functions/Startup.cs

```diff
@@ -44,10 +45,11 @@ public override void Configure(IFunctionsHostBuilder builder)
                 .Build();
 
             builder.Services.AddOptions();
-            builder.Services.Configure<AssessorApiAuthentication>(config.GetSection("AssessorApiAuthentication"));
-            
             builder.Services.Configure<AssessorApiAuthentication>(config.GetSection("AssessorApiAuthentication"));
             builder.Services.Configure<SqlConnectionStrings>(config.GetSection("SqlConnectionStrings"));
+
+            builder.Services.ConfigureHttpClients(config.GetSection("AssessorApiAuthentication").Get<AssessorApiAuthentication>(), configuration);
```

Comment:
This should be using something like `Services.AddHttpClient<AssessorServiceApiClient>()` and then putting the needed headers etc in AssessorServiceApiClient instead of creating your own HttpClients: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-3.0#typed-clients

Reply:
Ah would that would answer the above query then - I'll look into that